README.md

```diff
@@ -225,4 +220,25 @@ Each directory will be created with `cl-keytool.jar`, `cl-wallet.jar`, and a P12
 **In `code/data-l1`:**
 -   data-l1.jar     // The executable for the dL1 layer
 
+
+### `hydra remote_start`
+
+This method initiates the remote startup of your metagraph in one of the available networks: testnet, integrationnet, and mainnet.
+
+To commence the remote startup of the metagraph, we utilize the parameters configured on Euclid. The startup process unfolds as follows:
+
+1.  Termination of any processes currently running on the metagraph ports, which by default are 7000 for ml0, 8000 for cl1, and 9000 for dl1.
```

Comment:
Maybe add that these ports can be configured in the hosts.ansible file.

Reply:
It's already set there
``` 
  vars:
    ansible_ssh_common_args: "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
    metagraph_l0_public_port: 7000
    metagraph_l0_p2p_port: 7001
    metagraph_l0_cli_port: 7002
    currency_l1_public_port: 8000
    currency_l1_p2p_port: 8001
    currency_l1_cli_port: 8002
    data_l1_public_port: 9000
    data_l1_p2p_port: 9001
    data_l1_cli_port: 9002
```

I'll add this information on README